docs/source/admin/traffic_ops/using.rst

```diff
@@ -286,485 +274,110 @@ These are the types of servers that can be managed in Traffic Ops:
-|                |               |         | "consistent_hash"                                                                                                                                                        |
-+----------------+---------------+---------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| port           | parent.config | 80      | The port this parent is listening on as a forward proxy.                                                                                                                 |
-+----------------+---------------+---------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| use_ip_address | parent.config | 0       | 1 means use IP(v4) address of this parent in the parent.config, 0 means use the host_name.domain_name concatenation.                                                     |
-+----------------+---------------+---------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
-.. _qstring-handling:
-
-Qstring Handling
-----------------
-
-Delivery services have a Query String Handling option that, when set to ignore, will automatically add a regex remap to that delivery service's config.  There may be times this is not preferred, or there may be requirements for one delivery service or server(s) to behave differently.  When this is required, the psel.qstring_handling parameter can be set in either the delivery service profile or the server profile, but it is important to note that the server profile will override ALL delivery services assigned to servers with this profile parameter.  If the parameter is not set for the server profile but is present for the :term:`Delivery Service` profile, this will override the setting in the delivery service.  A value of "ignore" will not result in the addition of regex remap configuration.
+.. _asn-czf:
 
-+-----------------------+---------------+---------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|      Name             |    Filename   | Default |                                                                                    Description                                                                    |
-+=======================+===============+=========+===================================================================================================================================================================+
-| psel.qstring_handling | parent.config | -       | Sets qstring handling without the use of regex remap for 
```

Comment:
yeah, the content of that section needs to be relocated to more appropriate places. This PR  is but one step in that direction.

Also I'm not sure what you mean; this line was deleted.

Reply:
Okay, I think I understand now why you commented that. In the "Conversation" tab it was showing this as attached to a different, deleted line. The one with profile parameter for MSO.

This section wasn't _created_ by me, I just _moved_ it. I can't delete it because things reference it and replacing the content is beyond the scope of what I'm trying to accomplish.